app/bundles/IntegrationsBundle/Event/InternalContactProcessPseudFieldsEvent.php

```diff
@@ -0,0 +1,74 @@
+<?php
+
+declare(strict_types=1);
+
+/*
+ * @copyright   2020 Mautic Inc. All rights reserved
+ * @author      Mautic, Inc.
+ *
+ * @link        https://www.mautic.com
+ *
+ * @license     GNU/GPLv3 http://www.gnu.org/licenses/gpl-3.0.html
+ */
+
+namespace Mautic\IntegrationsBundle\Event;
+
+use Mautic\LeadBundle\Entity\Lead;
+use Symfony\Component\EventDispatcher\Event;
+
+class InternalContactProcessPseudFieldsEvent extends Event
```

Comment:
```suggestion
final class InternalContactProcessPseudoFieldsEvent extends Event
```
+ also fix the typo in the file name.